abci/client/client.go

```diff
@@ -74,22 +74,21 @@ type Callback func(*types.Request, *types.Response)
 
 type ReqRes struct {
 	*types.Request
-	*sync.WaitGroup
 	*types.Response // Not set atomically, so be sure to use WaitGroup.
 
-	mtx  sync.Mutex
-	done bool                  // Gets set to true once *after* WaitGroup.Done().
-	cb   func(*types.Response) // A single callback that may be set.
+	mtx    sync.Mutex
+	done   bool // Gets set to true once *after* WaitGroup.Done().
```

Comment:
There is no longer a WaitGroup, so I guess this should be "when signal is closed"?